server/controller/routes/house/getHouseByLocation.js

```diff
@@ -0,0 +1,22 @@
+const { getHouseLocation } = require('../../../database/queries/house');
+
+const getHouseByLocation = async (req, res, next) => {
+  try {
+    const { location } = req.query;
+    const { rows } = await getHouseLocation({ location });
+    if (rows.length > 0) {
```

Comment:
Sent a data ... if no data will send empty array 
You can handel this case on front